libkbfs/key_bundle_v3.go

```diff
@@ -214,6 +253,32 @@ func DeserializeTLFWriterKeyBundleV3(codec kbfscodec.Codec, path string) (
 	return wkb, nil
 }
 
+func tlfEphemeralPublicKeysSize(keys kbfscrypto.TLFEphemeralPublicKeys) int {
+	pkSize := int(unsafe.Sizeof(kbfscrypto.TLFEphemeralPublicKey{}))
```

Comment:
I guess we could ... although I don't know enough to really understand the risk of using `unsafe.Size` here. Could you elaborate?